src/combat.cpp

```diff
@@ -66,8 +66,11 @@ MatrixArea createArea(const std::vector<uint32_t>& vec, uint32_t rows)
 	return area;
 }
 
-std::vector<Tile*> getList(const MatrixArea& area, const Position& targetPos)
+std::vector<Tile*> getList(const MatrixArea& area, const Position& targetPos, const Direction dir)
 {
+	Position casterPos(targetPos.x, targetPos.y, targetPos.z);
+	casterPos = getNextPosition(dir, casterPos);
```

Comment:
```suggestion
	auto casterPos = getNextPosition(dir, targetPos);
```

Reply:
why `auto` and not `const Position`?

Reply:
It's personal, but it's shorter and just as clear. `getNextPosition` should return a `Position`, so why write it twice?